content/app-dev/create-actor/run.md

```diff
@@ -17,49 +17,69 @@ Building the actor is as easy as running `make` in the project's root directory.
+At this point, simply click **Start Actor** and then choose _From File_. Open a browser to your current project directory and then navigate to the `build` directory. Here you'll find a `(project)_s.wasm` file, where `(project)` is `hello`, or the name you selected for the project. The `_s` suffix indicates that this WebAssembly module has been _signed_.
 
 A moment later you should see the actor in your web UI as shown in the following screenshot:
 
 ![actor started](../actor.png)
 
 ### Start the web server
 
-We know our new actor needs a web server, so let's start the HTTP server capability provider. To do that, click the **Start Provider** button and enter this OCI URL: `wasmcloud.azurecr.io/httpserver:0.14.4`. Now we should have both an actor and a provider, and the dashboard should look like this:
+We know our new actor needs a web server, so let's start the HTTP server capability provider. To do that, click the **Start Provider** button and enter this OCI URL: `wasmcloud.azurecr.io/httpserver:0.14.6`. Now we should have both an actor and a provider, and the dashboard should look like this:
 
 ![actor and provider](../actorandprovider.png)
 
 ### Add a link definition
 
-With both the provider and the actor running, the next step is to _link_ the two. This provides a set of configuration values that is unique for each actor's use of a provider. To change things up slightly, and so you develop some muscle memory with the command line tooling, we'll use the `wash` CLI here. To link your actor, you'll' need the actor's public key. You can get that by clicking on the clipboard icon next to the actor name in the dashboard web UI (it'll be a long string that starts with **M**).
+With both the provider and the actor running, the next step is to _link_ the two. This provides a set of configuration values that is unique for each actor's use of a provider. To change things up slightly, and so you develop some muscle memory with the command line tooling, we'll use the `wash` CLI here. To link your actor, you'll' need the actor's public key. You can get that by clicking on the dark blue clipboard icon next to the actor name in the dashboard web UI (it'll be a long string that starts with **M**).
 
-Once you've got the actor's public key, you can enter the following command, replacing `ACTOR_ID` with the key you just put on your clipboard:
+Once you've got the actor's public key, you can go ahead and export a `HELLO_ACTOR_ID` environment variable with that value and then copy-and-paste the link command:
 
-```bash
-wash ctl link put ACTOR_ID VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver address=0.0.0.0:8087
+{{% tabs %}}
+{{% tab "Unix" %}}
+
+```shell
+# Paste your actor ID after the `=` below
```

Comment:
```
# Paste your actor ID after the `=` below (with no space after the `=`)
```

Reply:
Just noting this instruction is different for Windows, I'll add another comment to reflect that